src/docfx/SubCommands/MetadataCommand.cs

```diff
@@ -72,6 +72,7 @@ private MetadataJsonConfig ParseOptions(MetadataCommandOptions options, out stri
                 item.ShouldSkipMarkup |= options.ShouldSkipMarkup;
                 item.DisableGitFeatures |= options.DisableGitFeatures;
                 item.DisableDefaultFilter |= options.DisableDefaultFilter;
+                item.TocNamespaceStyle |= options.TocNamespaceStyle;
```

Comment:
It appears `|=` doesn't quite make sense here because `TocNamespaceStyle` is not a flags enum. The logic here is to override JSON config with command line options, so consider making `MetadataCommandOptions.tocNamespaceStyle` nullable and override when specified:

```suggestion
                item.TocNamespaceStyle = options.TocNamespaceStyle ?? item.TocNamespaceStyle;
```